libs/core/src/components/sage-button/test/sage-button.spec.tsx

```diff
@@ -58,7 +58,22 @@ describe('sage-button', () => {
     const svg = root.shadowRoot.querySelector('svg');
     expect(svg).not.toBeNull();
   });
-});
-
-
 
+  it('clicking on sage-button from inside a form calls `handleClick` method', async () => {
+    const root = await newSpecPage({
+      components: [SageButton],
+      html: `
+        <form id="#test">
+          <sage-button type="reset"></sage-button>
+        </form>  
+      `,
+    });
+    const form = root.doc.querySelector("form");
```

Comment:
I noticed you didn't specify a type on this `querySelector` but you didn't below.